src/main/java/duke/tasks/Event.java

```diff
@@ -0,0 +1,34 @@
+package duke.tasks;
+
+/**
+ * Class to create an event.
+ */
+class Event extends Task {
+
+    /**
+     * Constructor for Event.
+     *
+     * @param name text inputted by user for the Event.
+     */
+    public Event(String name) {
+        super(name);
+    }
+
+    /**
+     * returns string representation, with task type, marked, content and period.
+     *
+     * @return String representation of the Event.
+     */
+    @Override
+    public String toString() {
+        String checkBox;
+        char taskType1 = 'E';
```

Comment:
Similar to an above comment, would it better if the variable name was `taskTypeEvent` instead of `taskType1`?

```suggestion
        char taskTypeEvent = 'E';
```